### ISSUE TITLE
Someway to separate the list for podcast client

### ISSUE THREAD

**Issue by thejeshgn** [scripting/feedBase#23]
I have uploaded 

- My feedr eader opml list
- my podcast player list

It would be great to have someway to get separate some of them as podcast. So I can use that list with my podcast client. Else it will be filled with text feeds which makes mobile podcast client almost impossible to use.

Probably can be achieved through "lists" feature similar to Twitter? 
main.opml could be all subscriptions.

Thanks a ton. I am super excited.

**Comment by scripting** [comment]
Yes, I plan to have a way to create sub-lists, and main.opml, as you say will have all the subs.

I have the same use-case as you do. I have lots of lists, one for each tab on my reader page, and more. 

http://radio3.io/rivers/

There's also the podcatch.com list, which I make public through another mechanism.

https://github.com/scripting/Scripting-News/blob/master/podcatch/subscriptions.opml

I'd like to get all this into feedbase.io.

But first I wanted to let it settle in, and get some users involved. 

Dave
